contracts/bridges/BridgeTimelock.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../ITimelock.sol";
+
+contract BridgeTimelock is ITimelock {
```

Comment:
this Timelock no longer has any bridge-specific logic in it; we can probably rename it to just `Timelock` (or `TimelockBravo` to distinguish from the original?) and move it down to `contracts/`

Reply:
True. I don't have a strong opinion and am fine with either name.

Reply:
We should be able to get rid of `SimpleTimelock` at the same time (and I think we should do this to check assumptions and make sure things work as expected)

Reply:
going to remove `SimpleTimelock` as a follow-up; not going to add it to this PR (which has gotten rather large)